### PR TITLE
#1773 - Political correctness on user administration page

### PR DESCRIPTION
**Pull request by mbugert**
**What's in the PR**
* change checkbox label from "Show disabled users" to "Show disabled user accounts"

**How to test manually**
n/a

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

File: webanno-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/users/ManageUsersPage.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Copyright 2012
 # Ubiquitous Knowledge Processing (UKP) Lab and FG Language Technology
-# Technische Universit�t Darmstadt
+# Technische Universit�t Darmstadt
```
Comment (jcklie) [inline webanno-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/users/ManageUsersPage.properties:3]: This looks kaputt

**Review by jcklie — APPROVED** [review]

**Comment by reckart** [comment]
How about "Show disabled accounts" and leaving the ist out entirely?

**Comment by mbugert** [comment]
> How about "Show disabled accounts" and leaving the ist out entirely?

If you like that better, sure. I left "user" in because the whole tool refers to users as "users" and rarely as "accounts".

**Comment by reckart** [comment]
Jenkins, can you test this please?

**Pull request merged**